examples/engine-deploy.yml

```diff
@@ -0,0 +1,18 @@
+---
+- name: Setup ovirt repositories and deploy oVirt Engine
+  hosts: engine
+  vars_files:
+    - passwords.yml
+  vars:
+    ovirt_engine_setup_type: "ovirt-engine"
+    ovirt_engine_setup_version: "4.1"
+    ovirt_engine_setup_hostname: "localhost"
+    ovirt_engine_setup_organization: "example.com"
+    ovirt_engine_setup_dwh: true
+    ovirt_engine_setup_dwh_db_host: "localhost"
+    ovirt_engine_setup_configure_iso_domain: true
+    ovirt_engine_setup_firewall_manager: null
+    ovirt_repositories_ovirt_release_rpm: "http://plain.resources.ovirt.org/pub/yum-repo/ovirt-release41.rpm"
```

Comment:
Please change to 4.2 (right now it does not exist, but it should exist before this roles is officially released)

Reply:
Done